crates/sncast/tests/data/scripts/invoke/src/contract_does_not_exist.cairo

```diff
@@ -1,9 +1,14 @@
-use sncast_std::{invoke, InvokeResult};
+use sncast_std::{invoke, InvokeResult, ScriptCommandError, ProviderError, StarknetError};
 use starknet::{ContractAddress, Felt252TryIntoContractAddress};
 use traits::Into;
 
 fn main() {
+    println!("test");
```

Comment:
```suggestion
```

Reply:
I added `println!("test");` because of the weird behaviour of `.stdout_matches`

It seems that it's not possible to use `[..]` in the next line after `...`.

so for example, this will pass:
```
...
blabla 0x123
```

and this will fail:
```
...
blabla [..]
```

Reply:
we can use `assert_stdout_contains` from https://github.com/foundry-rs/starknet-foundry/pull/1763 after it's merged